tests/run-pass/fs.rs

```diff
@@ -215,6 +221,43 @@ fn test_symlink() {
     let mut contents = Vec::new();
     symlink_file.read_to_end(&mut contents).unwrap();
     assert_eq!(bytes, contents.as_slice());
+
+
+    #[cfg(unix)]
+    {
```

Comment:
We already have an `fs_libc.rs` test file, wouldn't these tests fit better there?

Reply:
This file already has some symlink setup code that I didn't want to duplicate.

Reply:
That's fair, but then the fs.rs vs fs_libc.rs separation doesn't really hold up any more.

Reply:
Both of those files have the same compilation flags - should I just merge them?

Reply:
Yeah seems reasonable.